test/e2e/csv_e2e_test.go

```diff
@@ -4247,6 +4248,20 @@ var _ = Describe("Disabling copied CSVs", func() {
 			return nil
 		}).Should(Succeed())
 		TeardownNamespace(ns.GetName())
+		Eventually(func() bool {
+			var namespaces corev1.NamespaceList
+			err := ctx.Ctx().Client().List(context.Background(), &namespaces,  &client.ListOptions{})
+			if err != nil || len(namespaces.Items) == 0{
+				return false
+			}
+			for _, n := range namespaces.Items {
+				if ns.GetName() == n.GetName() {
+					return false
+				}
+			}
+
+			return true
```

Comment:
Can we use the c-r dynamic client instead and fire off a GET query for this specific namespace so we don't have to iterate over the entire set of cluster namespaces? We can then return an error and check whether that error is a 404 (e.g. `apierrors.IsNotFound(err)`).

Reply:
I think you can do something like the following (I scribbled this inline so YMMV):

```go
		Eventually(func() error {
			var namespace corev1.Namespace
			return ctx.Ctx().Client().Get(context.Background(), client.ObjectKeyFromObject(&ns), &namespace)
		}).Should(WithTransform(k8serrors.IsNotFound, BeTrue()))
```

WDYT?

Reply:
I'll try.  Thanks!